sdks/java/io/tika/src/main/java/org/apache/beam/sdk/io/tika/TikaOptions.java

```diff
@@ -40,31 +40,6 @@
   String getContentTypeHint();
   void setContentTypeHint(String value);
 
-  @Description("Metadata report status")
```

Comment:
I suppose a next step would be to remove support for this Options class.

Reply:
I'd like to suggest reviewing it once the configuration support at the API level is in place, perhaps it may stilll prove useful for users...